bin/dotd

```diff
@@ -236,6 +237,26 @@ def print_dtp_message
   end
 end
 
+# Retrieves recent Honeybadger issues, displaying them on the console.
+def get_honeybadger_issues
+  issues = Honeybadger.get_recent_issues
```

Comment:
Would it be useful to sort these by environment? By project (cronjobs, dashboard, pegasus)?